tests/utils/playwright/datalens/datalens-test-setup.ts

```diff
@@ -44,15 +42,7 @@ export async function datalensTestSetup({config, afterAuth, authSettings}: Datal
     }
 
     try {
```

Comment:
Should we remove afterAuth as well with all try/catch? It seems redundant without authentication